lib/grant.js

```diff
@@ -345,7 +345,7 @@ function useMfaOtpGrant (done) {
           self.req.body.expires_in = user.expires_in
       }
 
-      if (self.req.body && user.expires_at) {
+      if (self.req.body && user.expires_at !== undefined) {
```

Comment:
I realize that this is probably an artifact of how the library works, but seeing `user.expires_at` looks really weird.

Reply:
Agreed.